pdp/static/js/pdp_raster_map.js

```diff
@@ -53,25 +87,31 @@ function dasToUnitsSince(data) {
         m = reg.exec(s),
         units = m[1],
         dateString = m[3],
-        sDate;
-
+        sDate;   
+    
+    var calendar;
+    reg = /calendar \"(standard|365_day|360_day)\"/,
+    m = reg.exec(s),
+    calendar = m[1];
+        
     reg = /(\d{4})-(\d{1,2})-(\d{1,2})( |T)(\d{1,2}):(\d{1,2}):(\d{1,2})/g;
     m = reg.exec(dateString);
     if (m) {
         sDate = new Date(m[1], parseInt(m[2], 10) - 1, // Months in das result are 1-12, js needs 0-11
                          m[3], m[5], m[6], m[7], 0);
-        return [units, sDate];
+        return [units, sDate, calendar];
```

Comment:
Yeah, this works to add another item to the list. However, if we end up hacking in this code *much* more than this, we'll probably want to convert the return value to be a properly attributed key/value mapping (rather than having a list of items that have a particular meaning without any context).